shader/draw_leaf.wgsl

```diff
@@ -169,6 +171,21 @@ fn main(
                 info[di + 9u] = bitcast<u32>(ra);
                 info[di + 10u] = bitcast<u32>(roff);
             }
+            // DRAWTAG_FILL_IMAGE
+            case 0x248u: {
+                info[di] = bitcast<u32>(linewidth);
+                let inv_det = 1.0 / (matrx.x * matrx.w - matrx.y * matrx.z);
```

Comment:
It would be great to do a pass through the code to make use of appropriate matrix types/ops where appropriate. For this PR, I'd like to avoid the extra churn though.

Reply:
sgtm!